packages/decoder/lib/decoders/contract.ts

```diff
@@ -612,13 +611,16 @@ export class ContractInstanceDecoder {
    */
   public watchMappingKey(variable: number | string, ...indices: any[]): void {
```

Comment:
Hey you forgot to make `watchMappingKey` and `unwatchMappingKey` async like we talked about :P

Reply:
ah cool, will do!

Reply:
(Note obviously this will also require modifying tests appropriately!)